### DISCUSSION TITLE
部署完之后页面502

### DISCUSSION THREAD

**Discussion by lbrsy** [1Panel-dev/1Panel#6095] [General]
系统Ubuntu 24.04 LTS

**Comment by lbrsy** [comment]
![1723435980793](https://github.com/user-attachments/assets/46348b8f-5ac5-4af3-9a84-b8ab608acbd8)

**Reply by wanghe-fit2cloud** [reply]
1Panel 服务运行正常，就先检查防火墙和安全组配置是否正确。

**Comment by lbrsy** [comment]
![image](https://github.com/user-attachments/assets/9c147043-84fb-482f-993b-30d3ada51103)
